README.md

```diff
@@ -24,38 +33,46 @@ A commit node represents a commit in git log history. Commits are connected to o
 
 A file node represents a file in git repository. Every commit is connected to own files, but file also can be connected with commits which touched (added, removed, or modified) the file.
 
-### Usage
+### usage
 
 This PoC exposes an API and following tools:
 * import - lets you import a git repository into graph database (backed by cayley.io).
-```
-codegraph git import [options] <repo>
-  -db string
-    	database directory (default ".")
+```bash
+$   ./codegraph git import <repo> [<repos>...] [flags]
```

Comment:
Since we use `go install`, it should be okay to run it as `codegraph` (assuming `$GOPATH/bin` is added to `PATH`).